releases/istio-gatekeeper.yaml

```diff
@@ -0,0 +1,303 @@
+  ## https://github.com/helm/charts/tree/master/incubator/raw                   #
+  ###############################################################################
+- name: 'istio-gatekeeper-additional'
+  chart: "kubernetes-incubator/raw"
+  namespace: "istio-system"
+  labels:
+    component: "iap"
+    namespace: "istio-system"
+    default: "false"
+  version: "0.1.0"
+  wait: true
+  force: true
+  recreatePods: true
+  installed: {{ env "ISTIO_GATEKEEPER_INSTALLED" | default "false" }}
+  values:
+  - resources:
+    - apiVersion: networking.istio.io/v1alpha3
+      kind: Gateway
+      metadata:
+        name: istio-oidc-ingressgateway
+      spec:
+        selector:
+          istio: istio-oidc-ingressgateway
+        servers:
+          - hosts:
+              - "*"
+            port:
+              name: http
+              number: 80
+              protocol: HTTP
+
+  ##############################################################################
+  ## Forecastle ################################################################
+  ## Forecastle is a dynamic web portal for Kubernetes services.              ##
+  ## Used here to provide a page listing all the components deployed behind   ##
+  ## the OIDC ingress.                                                        ##
+  ##############################################################################
+  #
+  # References:
+  #   - https://github.com/stakater/Forecastle/tree/v1.0.27/deployments/kubernetes/chart/forecastle
+  # v1.0.27  url: "git+https://github.com/stakater/Forecastle@deployments/kubernetes/chart?ref=ffb5179aa659e0e3cf0ca20e1c8bd94c5fd66a2e"
+  #   - https://github.com/stakater/Forecastle
+  #
+- name: "istio-gatekeeper-forecastle"
+  namespace: "istio-system"
+  labels:
+    chart: "forecastle"
+    repo: "github"
+    component: "iap"
+    namespace: "istio-system"
+    vendor: "stakater"
+    default: "false"
+  chart: "forecastle/forecastle"
+  version: "v1.0.27"
+  wait: false
+  installed: {{ env "ISTIO_GATEKEEPER_FORECASTLE_INSTALLED" | default "true" }}
+  force: true
+  recreatePods: true
+  values:
+    - nameOverride: istio-gatekeeper-forecastle
+      fullNameOverride: istio-gatekeeper-forecastle
+      forecastle:
+        image:
+          pullPolicy: "IfNotPresent"
+        namespace: "oidc-ingress"
+        deployment:
+          # The chart sets lots of annotations by default unless explicitly cleared
+          annotations: ""
+          replicas: '{{- env "ISTIO_GATEKEEPER_FORECASTLE_REPLICAS" | default "2" -}}'
+          revisionHistoryLimit: '{{- env "ISTIO_GATEKEEPER_FORECASTLE_REVISION_HISTORY_LIMIT" | default "10" -}}'
+        config:
+          # Let the Forecastle instance name reflect the nginx ingress class
+          # for simplicity and ease of understanding.
+          instanceName: '{{ env "ISTIO_GATEKEEPER_FORECASTLE_NGINX_CLASS" | default "mock-ingress" }}'
```

Comment:
`ISTIO_GATEKEEPER_FORECASTLE_NGINX_CLASS` should have the same default here and on line 275 below. Probably `oidc-ingress` or `istio-oidc-ingress` is less confusing than `mock-ingress`. 

Remember that the `instanceName` is going to be used by other apps to designate where they want their links displayed. Since it appears the value of `kubernetes.io/ingress.class` (below) is ignored if there is no matching `nginx` controller, it might be even better to use `oidc-portal` in both places.